README.md

```diff
@@ -17,35 +15,89 @@ This library provides the VarDumper helper.
 [![static analysis](https://github.com/yiisoft/var-dumper/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/var-dumper/actions?query=workflow%3A%22static+analysis%22)
 [![type-coverage](https://shepherd.dev/github/yiisoft/var-dumper/coverage.svg)](https://shepherd.dev/github/yiisoft/var-dumper)
 
-### Unit testing
+`VarDumper` enhances functionality of `var_dump()`, `print_r()`, and `var_export()`. It is dealing with recursive references,
+may highlight syntax, export closures and output as JSON.
+
+## General usage
+
+### Quick debugging
+
+In case you want to echo a string representing variable contents use the following code:
+
+```php
+\Yiisoft\VarDumper\VarDumper::dump($variable, 10, true);
+```
+
+That is useful for quick debugging. `10` is maximum recursion depth and `true` is telling dumper to
+highlight syntax.
+
+### Getting a string
+
+To get a string representing variable contents, same as above but without `echo`:
+
+```php
+$string = \Yiisoft\VarDumper\VarDumper::create($variable)->asString(10, true);
+```
+
+`10` is maximum recursion depth and `true` is telling dumper to
+highlight syntax.
+
+### Exporting as PHP code
+
+In order to get a valid PHP expression string that can be evaluated by PHP parser,
+and the evaluation result will give back the variable value, use the following code:
+
+```php
+$string = \Yiisoft\VarDumper\VarDumper::create($variable)->export();
+$string = \Yiisoft\VarDumper\VarDumper::create($variable)->asPhpString();
```

Comment:
These two are almost equal. Instead of two methods we may add a `prettyPrint` flag to `export()` or `asPhpString()` similar to JSON and remove second method. Or we can introduce more methods to JSON. Should be consistent either way.